src/platforms/ios/ioscontroller.swift

```diff
@@ -237,10 +237,21 @@ public class IOSControllerImpl : NSObject {
         TunnelManager.session?.stopTunnel()
     }
 
+    @objc func removeTunnel() {
```

Comment:
I think we're renaming this function too, right? I just wanted to point out I see both `deleteTunnel` and `removeTunnel` in the changes

Reply:
Yes, thanks. I just updated both `removeTunnel` and `deleteTunnel` to `deleteOSTunnelConfig`.